deploy/apecloud-mysql/config/mysql8-config-constraint.cue

```diff
@@ -187,7 +187,7 @@
 	default_password_lifetime: int & >=0 & <=65535 | *0
 
 	// The default storage engine (table type).
-	default_storage_engine?: string & "InnoDB" | "MRG_MYISAM" | "BLACKHOLE" | "CSV" | "MEMORY" | "FEDERATED" | "ARCHIVE" | "MyISAM" | "xengine" | "XENGINE" | "INNODB" | "innodb"
```

Comment:
since this is not a fix, is backward compatible handling required? and what is minimum required ApeCloud MySQL version?

Reply:
good idea 👍